todo-list/client/src/components/Button/index.jsx

```diff
@@ -0,0 +1,11 @@
+import { Button, } from "@mui/material"
+import { LanguageIcon } from '@mui/icons-material/Language';
+
+
+export const ButtonComponent = ({variant,text,color,icon})=>{
+    return(
+        <Button size="large" variant = {variant} color = {color} sx={{borderRadius:'12px'}} > {icon} {text}</Button>
```

Comment:
We discussed that you don't need to use spaces in props. Example - `variant={variant}`